CMakeLists.txt

```diff
@@ -685,6 +692,10 @@ if (FILAMENT_BUILD_FILAMAT OR IS_HOST_PLATFORM)
     if (FILAMENT_ENABLE_MATDBG OR IS_HOST_PLATFORM)
         add_subdirectory(${LIBRARIES}/matdbg)
     endif()
+
+    if (FILAMENT_ENABLE_FRAMEGRAPH OR IS_HOST_PLATFORM)
```

Comment:
Where is `FILAMENT_ENABLE_FRAMEGRAPH` defined? Do you mean `FILAMENT_ENABLE_FGDBG`?

Reply:
Good catch! Yes, I renamed the project and missed this one, thank you!